snap/snapcraft.yaml

```diff
@@ -1,6 +1,7 @@
 name: tailscale
 base: core22
 version: 1.58.2
```

Comment:
Not related to this PR, but why are we fixing a version with more than one year of [release](https://github.com/tailscale/tailscale/releases) (current is on 1.72.1)?

I think we need to change this approach for new builds use the latest tags and we promote to other channels eventually. This is more or less related with #3

Reply:
Yep this is the very first initial version; we plan to iterate on it. :)